variables.tf

```diff
@@ -288,6 +288,12 @@ variable "alb_ingress_port" {
   default     = 443
 }
 
+variable "alb_redirect_port" {
```

Comment:
Does it makes sense to have a variable here if the protocol is pinned to "HTTP"?